src/explore-education-statistics-admin/src/components/editable/Comments.tsx

```diff
@@ -133,21 +144,73 @@ const Comments = ({
 
         <ul className={styles.commentsContainer}>
           {orderBy(comments, ['created'], ['desc']).map(comment => {
-            const { createdBy } = comment;
+            const { createdBy, resolvedBy } = comment;
 
-            return (
+            return comment.resolved ? (
+              <li className="govuk-body-s">
+                <p className="govuk-!-margin-0">
+                  <strong>Comment resolved</strong>
+                </p>
+                <p className="govuk-!-margin-0">
+                  <FormattedDate format="dd/MM/yy HH:mm">
```

Comment:
Not a big fan of this date format as it's bit too dense on the eyes and is also ambiguous. Would suggest we use the old date format, but with shorter months: `d MMM yyyy HH:mm`.

Same applies for the other date/times here

Reply:
The reason I went for this is because there is limited width for text in the comments - the longer date was wrapping which I think is more confusing than the shorter date. I was even considering getting rid of the time, but decided against that.

I'll run both by Laura and go with what she wants.

Reply:
Laura says she prefers the first so I'm going to leave this.

Reply:
Cool, fair enough!